content/_releases/22/36.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Product Update - v22.36
+description: Details of the product update v22.36 from 8th of September 2022.
+layout: rel
+version: "22.36"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-09-08
+redirect_from:
+  - /releases/22.36.html
+---
+
+## Improvements and Updates
+
+### Credentials page: new design
+
+This is continuation of our effort to modernise and harmonise look and feel of the
+platform user interface. In this release we improved the Credentials page. When
+you navigate to Integrate > Credentials page you will see a similar view:
+
+{% include img.html max-width="80%" url="/assets/img/RN/2236/RN2236-credentials-page.png" title="Credentials Page" %}
+
+When you select a component your screen would look like the one below for Salesforce:
+
+{% include img.html max-width="80%" url="/assets/img/RN/2236/RN2236-credential.png" title="Component Credential" %}
+
+You can select 3 dots to open the menu to **Verify**, **Edit** or **Delete** the credentials
+or use **+** (plus) in the right above to create a new one.
+
+{% include img.html max-width="80%" url="/assets/img/RN/2236/RN2236-credential-mod.png" title="Credential Modification" %}
+
+Here is how each credential page looks when you Edit it:
```

Comment:
If nothing changed here I suggest to remove the Edit Credential section (image + Line 32)